k2qc/core.py

```diff
@@ -120,7 +127,12 @@ def verify_positive_flux(self):
             warnings.simplefilter("ignore")  # ignore NaN comparison warnings
             mask = self.tpf[1].data['QUALITY'] == 0
             flux_plus_bkg = self.tpf[1].data['FLUX'][mask] + self.tpf[1].data['FLUX_BKG'][mask]
-            assert (flux_plus_bkg < 0).sum() == 0
+            if (flux_plus_bkg < 0).sum() > 0:
+                badpixels = np.argwhere(flux_plus_bkg < 0)
+                for pix in badpixels:
+                    print("Negative flux values found at frame {},"
```

Comment:
Can we make it say "(FLUX + FLUX_BKG) < 0" instead of "Negative flux values", just to be more explicit?